tools/files.js

```diff
@@ -177,12 +181,13 @@ files.getToolsVersion = function () {
 // Return the root of dev_bundle (probably /usr/local/meteor in an
 // install, or (checkout root)/dev_bundle in a checkout.).
 files.getDevBundle = function () {
-  return path.join(files.getCurrentToolsDir(), 'dev_bundle');
+  return files.pathJoin(files.getCurrentToolsDir(), 'dev_bundle');
 };
 
 // Return the top-level directory for this meteor install or checkout
 files.getCurrentToolsDir = function () {
-  return path.join(__dirname, '..');
+  var dirname = convertToStandardPath(__dirname);
+  return files.pathJoin(dirname, '..');
```

Comment:
need to make sure this call works correctly on windows given all our back-and-forth converts